source/common/runtime/runtime_features.cc

```diff
@@ -61,6 +61,7 @@ constexpr const char* runtime_features[] = {
     "envoy.reloadable_features.reject_unsupported_transfer_encodings",
     // Begin alphabetically sorted section.
     "envoy.deprecated_features.allow_deprecated_extension_names",
+    "envoy.reloadable_features.disallow_dns_cache_circuit_breakers",
```

Comment:
These are true by default, so I think you want `envoy.reloadable_features.enable_dns_cache_circuit_breakers`